admin/common_auth/views.py

```diff
@@ -77,6 +77,11 @@ def form_valid(self, form):
         prereg_admin_group = Group.objects.get(name='prereg_admin')
         for group in form.cleaned_data.get('group_perms'):
             osf_user.groups.add(group)
+            group_type = group.name.split('_')[0]
```

Comment:
Gut says this is the wrong place to do this. Above it sets the user to staff. Are all preprint provider moderators set as staff?

Reply:
It looks like all preprint provider moderators who are added through the admin interface are set to staff (not osf_admin though). They don't need to be set to staff though.